test/razorpay/test_transfer.rb

```diff
@@ -0,0 +1,73 @@
+require 'test_helper'
```

Comment:
Why this file?

Reply:
because it has mock request functions like `stub_get` `stub_post` `stub_put` `stub_fetch` `stub_put` & `stub_delete` and its required while testing

Reply:
No. I asked why `test_transfer.rb` in this PR.

Reply:
i have removed `test_transfer.rb`